src/domain/customers/utils.ts

```diff
@@ -14,12 +14,12 @@ export const getCustomersData = (data: CUSTOMERS | undefined | any) => {
     return data.profiles.edges.map(profile => {
       if (profile?.node) {
         const profileData: CustomerData = {
-          email: profile.node.emails[0]?.email,
+          email: profile.node.emails?.[0]?.email,
           firstName: profile.node.firstName,
           id: profile.node.id,
           lastName: profile.node.lastName,
           nickname: profile.node.nickname,
-          phone: profile.node.phones[0]?.phone,
+          phone: profile.node.phones?.[0]?.phone,
         };
         return profileData;
       }
```

Comment:
Can we remove `any` from the function's argument type and `email`, `phone` from `CustomerData` interface?

Reply:
Yeah